app/views/shared/_sequence_next_prev.html.haml

```diff
@@ -3,11 +3,11 @@
 .intra-sequence-nav
   .next-activity{ :class => ('disabled' if next_activity.nil?) }
     - if next_href
-      = link_to 'Next Activity >', sequence_activity_path(sequence, next_activity), { :title => next_activity.name }
+      = link_to 'Next Activity >', runnable_activity_path(sequence, next_activity), { :title => next_activity.name }
     - else
       Next Activity >
   .last-activity{ :class => ('disabled' if previous_activity.nil?) }
     - if last_href
-      = link_to "< Last Activity", sequence_activity_path(sequence, previous_activity), { :title => previous_activity.name }
+      = link_to "< Last Activity", runnable_activity_path(sequence, previous_activity), { :title => previous_activity.name }
```

Comment:
This looks broken. `runnable_activity_path` doesn't seem to take a sequence as a param.

Reply:
Right, [`runnable_activity_path`](https://github.com/concord-consortium/lara/blob/master/app/helpers/lightweight_activity_helper.rb#L12) doesn't accept a sequence.  Sort of weird that this didn't blow up more noticeably.

`runnable_activity_path` It depends on an instance variable `@sequence` being set in the [app controller](https://github.com/concord-consortium/lara/blob/master/app/controllers/application_controller.rb#L108)

Oops.